servant-client/servant-client.cabal

```diff
@@ -53,6 +53,7 @@ library
     , text                  >= 1.2      && < 1.3
     , transformers          >= 0.3      && < 0.6
     , transformers-compat   >= 0.4      && < 0.6
+    , http-conduit
```

Comment:
Definitely not worth this dependency for the `HasManager` instance. Sorry! I didn't know it didn't exist previously!